requirements/common.txt

```diff
@@ -17,6 +17,7 @@ pillow  # Required for image processing
 prometheus-fastapi-instrumentator >= 7.0.0
 tiktoken >= 0.6.0  # Required for DBRX tokenizer
 lm-format-enforcer >= 0.10.11, < 0.11
+llguidance>=0.6.15
```

Comment:
I think we need to restrict this to x86 and arm https://pypi.org/project/llguidance/#files